devel/patchelf/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                patchelf
+version             0.10.0
+categories          devel
+platforms           darwin
+license             GPL-3
+maintainers         @lkollar
```

Comment:
please add an obfuscated e-mail address here as well, and ideally also `openmaintainer` so that minor changes can be made without you approving them.